docs/user_guide/assessment.rst

```diff
@@ -1,20 +1,134 @@
+
+* In a job screening scenario, we need to quantify the number of candidates that are classified as "negative" (not recommended for the job), but whose true label is "positive" (they are "qualified"). One possible metric is the false negative rate: fraction of qualified candidates that are screened out. Note that before we attempt to classify candidates, we need to determine the construct validity of the "qualified" status; more information on construct validity can be found in :ref:`construct_validity`.
+
+* For a speech-to-text application, the harm could be measured by disparities in the word error rate for different group, measured by the number of mistakes in a transcript divided by the overall number of words.
+
+Note that in some cases, the outcome we seek to measure is not 
+directly available. 
+Occasionally, another variable in our dataset provides a close 
+approximation to the phenomenon we seek to measure. 
+In these cases, we might choose to use that closely related variable, 
+often called a "proxy", to stand in for the missing variable. 
+For example, suppose that in the job screening scenario, 
+we have data on whether the candidate passes the first two stages, 
+but not if they are ultimately recommended for the job. 
+
+As an alternative to the unobservable final recommendation, we could
+therefore measure the harm using the proxy variable indicating whether
+the candidate passes the first stage of the screen.
+If you choose to use a proxy variable to 
+represent the harm, check the proxy variable regularly to ensure it 
+remains useful over time. Our section on :ref:`construct_validity`
+describes how to determine whether a  
+proxy variable measures the intended construct in a meaningful 
+and useful way. It is important to ensure that the proxy is suitable 
+for the social context of the problem you seek to solve. 
+In particular, be careful of falling into one of the :refabstraction_traps. 
+
+Compare quantified harms across the groups
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The workhorse of fairness assessment in Fairlearn are disaggregated metrics, which are 
+metrics evaluated on slices of data. For example, to measure harms due to 
+errors, we would begin by evaluating the errors on each slice of the data that 
+corresponds to a group. If some of the groups are seeing much larger errors 
+than other groups, we would flag this as a fairness harm.
+
+To summarize the disparities in errors (or other metrics), we may want to 
+report quantities such as the difference or ratio of the metric values between 
+the best and the worst slice. In settings where the goal is to guarantee 
+certain minimum quality of service across all groups (such as speech recognition), 
+it is also meaningful to report the worst performance across all considered groups.
+
+For example, when comparing the false negative rate across groups defined by race, 
+we may summarize our findings with a table like the following:
```

Comment:
Having worked with these for a while now I don't trust any such tables anymore because they don't tell me the size of each group. Perhaps we should have an example that has that, too?

Reply:
I can add some hypothetical group population counts, if that would work?

Reply:
I added the population counts as well as the warning "Note that the these statistics must 
be drawn from a large enough sample size to draw meaningful conclusions.".